packages/contracts/contracts/SmartEscrowMono.sol

```diff
@@ -13,22 +13,22 @@ import "./IArbitrator.sol";
 interface IWETH {
   // brief interface for canonical ether token wrapper contract
   function deposit() external payable;
-
-  function transfer(address dst, uint256 wad) external returns (bool);
 }
 
 // splittable digital deal lockers w/ embedded arbitration tailored for guild work
 contract SmartEscrowMono is Context, IArbitrable, ReentrancyGuard {
   using SafeMath for uint256;
   using SafeERC20 for IERC20;
 
-  uint256 internal constant DISPUTES_POSSIBLE_OUTCOMES = 3;
+  uint256 internal constant DISPUTES_POSSIBLE_OUTCOMES = 5; // excludes options 0, 1 and 2
```

Comment:
These options are excluding the default 3 options. 
Aragon expects us to send on the number of "extra" ruling options we require.

Reply:
OK cool - I think those splits are good for now (and it doesn't seem too difficult to add them in the future)